### PR TITLE
chore(version): upgrading to oss 1.33.x

### PR DESCRIPTION
**Pull request by nemesisOsorio**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,8 +1,7 @@
-korkVersion=7.206.0
+korkVersion=1-33.1-nemesis-SNAPSHOT
```
Comment (nemesisOsorio) [inline gradle.properties:1]: once [kork](https://github.com/OpsMx/kork-oes/pull/130) is ready we can change this version

File: rosco-web/rosco-web.gradle [hunk 2]
```diff
@@ -3,20 +3,22 @@ apply plugin: 'io.spinnaker.package'
...
+  implementation "io.spinnaker.kork:kork-swagger"
 
-  implementation "org.codehaus.groovy:groovy"
+  implementation "org.apache.groovy:groovy"
   implementation "io.spinnaker.kork:kork-artifacts"
   implementation "io.spinnaker.kork:kork-config"
   implementation "org.springframework.boot:spring-boot-starter-web"
+    implementation "org.springframework.boot:spring-boot-starter-security"
+    implementation "org.springframework.security:spring-security-core"
+    compileOnly "org.springframework.boot:spring-boot-starter-actuator"
```
Comment (rahul-chekuri) [inline rosco-web/rosco-web.gradle:21]: nitpick: Indentation

File: rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/persistence/config/JedisConfig.groovy [hunk 3]
```diff
@@ -59,12 +59,19 @@ class JedisConfig {
         } finally {
           jedis?.close()
         }
-        def internal = jedisPool.internalPool //thx groovy
+	//commented for exception: groovy.lang.MissingPropertyException:
+        // No such property: internalPool for class: redis.clients.jedis.JedisPool
+        /* def internal = jedisPool.internalPool //thx groovy
```
Comment (rahul-chekuri) [inline rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/persistence/config/JedisConfig.groovy:64]: cleanup this commented out code

File: rosco-core/rosco-core.gradle [hunk 4]
```diff
@@ -13,11 +13,13 @@ dependencies {
   implementation "io.spinnaker.kork:kork-web"
   implementation "com.squareup.retrofit2:converter-jackson"
   implementation "com.squareup.retrofit2:retrofit"
-  implementation "io.reactivex:rxjava"
+  implementation "io.reactivex:rxjava:1.3.8"
```
Comment (rahul-chekuri) [inline rosco-core/rosco-core.gradle:16]: why are we specific about the version here?

**Review by rahul-chekuri — COMMENTED** [review]
@aman-agrawal please update the PR with the things requested.

**Comment by aman-agrawal** [comment]
> @aman-agrawal please update the PR with the things requested.

This is Nemesis PR, he took different way of doing rebasing, you may not find commits over here as discussed in today's standup.

File: rosco-core/rosco-core.gradle (hunk 4 above)
Comment (aman-agrawal) [reply rosco-core/rosco-core.gradle:16]: coming from rosco-oes:1.30.1 : [ref](https://github.com/OpsMx/rosco-oes/blob/a82eb4e6d996745e3427072a28cdc7250defae81/rosco-core/rosco-core.gradle#L17)

File: rosco-web/rosco-web.gradle (hunk 2 above)
Comment (aman-agrawal) [reply rosco-web/rosco-web.gradle:21]: done

File: rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/persistence/config/JedisConfig.groovy (hunk 3 above)
Comment (aman-agrawal) [reply rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/persistence/config/JedisConfig.groovy:64]: part of OSS code

**Comment by aman-agrawal** [comment]
> @aman-agrawal please update the PR with the things requested.

Answered

**Review by rahul-chekuri — APPROVED** [review]

File: rosco-web/rosco-web.gradle (hunk 2 above)
Comment (aman-agrawal) [reply rosco-web/rosco-web.gradle]: done

File: gradle.properties (hunk 1 above)
Comment (aman-agrawal) [reply gradle.properties:1]: https://github.com/OpsMx/rosco-oes/commit/c2f75dc777f662f5122e264dcf462a167887acae

**Pull request merged**